BDArmory/ModuleWeapon.cs

```diff
@@ -1063,24 +1074,12 @@ private void Fire()
                         firedBullet.transform.position = fireTransform.position;
 
                         pBullet.caliber = bulletInfo.caliber;
-                        pBullet.bulletVelocity = bulletInfo.bulletVelocity;
                         pBullet.bulletMass = bulletInfo.bulletMass;
                         pBullet.explosive = bulletInfo.explosive;
                         pBullet.apBulletMod = bulletInfo.apBulletMod;                  
                         pBullet.bulletDmgMult = bulletDmgMult;
-
-                        //A = π x (Ø / 2)^2
-                        bulletDragArea = Mathf.PI * Mathf.Pow(caliber / 2f, 2f);
-
-                        //Bc = m/Cd * A
-                        bulletBallisticCoefficient = bulletMass / ((bulletDragArea / 1000000f) * 0.295f); // mm^2 to m^2
-                        
-                        //Bc = m/d^2 * i where i = 0.484
-                        //bulletBallisticCoefficient = bulletMass / Mathf.Pow(caliber / 1000, 2f) * 0.484f;
-
```

Comment:
moved from Fire() to Start(), so the ballistic coefficient is known without firing the gun